.github/workflows/runas.yml

```diff
@@ -0,0 +1,39 @@
+name: Test cargo pgrx test --runas
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: 1
+  RUSTFLAGS: "-C link-args=-Wl,-undefined,dynamic_lookup"   # we don't want target-cpu=native from `.cargo/config` b/c we're caching binaries but we want the other stuff
```

Comment:
:raised_eyebrow: `-Ctarget-cpu=native` is not in our repo's `.cargo/config.toml`?